provider/openstack/provider.go

```diff
@@ -1275,11 +1275,12 @@ func (e *Environ) startInstance(
 
 	server, err := tryStartNovaInstance(shortAttempt, e.nova(), opts)
 	if err != nil || server == nil {
-		err := errors.Annotate(err, "cannot run instance")
+		logger.Debugf("cannot run instance full error: %q", err)
+		errCause := errors.Cause(err)
+		err := errors.Annotate(errCause, "cannot run instance")
```

Comment:
You can remove the `:` to avoid shadowing `err`

Reply:
done